packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -25,8 +35,56 @@ export default defineConfig({
   requestTimeout: 30000,
   e2e: {
     // @ts-ignore
-    setupNodeEvents(on, config) {
-      config.env.ADDRESS = process.env.ADDRESS
+    async setupNodeEvents(on, config) {
+      const wallet = new Wallet(process.env.PRIVATE_KEY_LOCAL!)
```

Comment:
nit: let's revert back to `PRIVATE_KEY` which is in the `.e2e.env.sample`

Reply:
As discussed on Slack, changed to `PRIVATE_KEY_CUSTOM` and added a comment why we use that